src/Preference.cpp

```diff
@@ -9,9 +9,10 @@
 
 static SubscriptionManager<IPreference> m_Subscribers;
 
-IPreference::IPreference( const RString& sName ):
+IPreference::IPreference( const RString& sName, PreferenceFlag flags ):
```

Comment:
With the other comment, `PreferenceType type`

Reply:
done!